src/dat/gui/GUI.js

```diff
@@ -557,6 +561,27 @@ common.extend(
     },
 
     /**
+     * Adds a new custom controller to the GUI.
+     *
+     * @param object
+     * @param property
+     * @returns {Controller} The controller that was added to the GUI.
+     * @instance
+     *
+     */
+    addCustomController: function(object, property) {
```

Comment:
To keep this simple, maybe let's only support `gui.add( controller )` for now?

Reply:
The addCustomController function is the easiest way of using of custom controller. It is ready and I do not want to remove it.

Reply:
See https://github.com/dataarts/dat.gui/pull/232#discussion_r289620635 – I think that `add` can be just as easy, but if not could you share examples showing how to use them both? It doesn't seem to me like we need both, if `add` can be simplified.

Reply:
I have removed addCustomController from my code